dbms/src/Server/StorageConfigParser.cpp

```diff
@@ -444,9 +446,135 @@ std::tuple<size_t, TiFlashStorageConfig> TiFlashStorageConfig::parseSettings(
         storage_config.remote_cache_config.parse(config.getString("storage.remote.cache"), log);
     }
 
+    if (config.has("storage.tmp"))
```

Comment:
maybe use `storage.temp` instead of `storage.tmp`?

Reply:
Why `temp` is better? Becase I use `storage.tmp` to keep align with original `tmp_path`.

Reply:
tidb, tikv both use "temp" instead of "tmp", though one is "temp-dir" and the other is "temp-path"..

https://docs.pingcap.com/tidb/stable/tidb-configuration-file/#temp-dir-new-in-v630
https://docs.pingcap.com/tidb/stable/tikv-configuration-file/#temp-path-new-in-v620

Reply:
if the user setup "temp directory" on tidb, tikv, tiflash. It is odd to have "temp-dir"/"temp-path"/"tmp". If we use "storage.temp", at least the term "temp" is aligned.

Reply:
looks like tidb has `tmp-storage-path` and `temp-dir` (the former one is for spilling file, the later one is for index creating).

Reply:
@guo-shaoge That is a bad :(
@joey-yez Please confirm whether tiflash should use "storage.tmp.dir" or "storage.temp.dir"?

Reply:
After discussion with @joey-yez , we decide use `temp`